apps/web/app/constants.ts

```diff
@@ -0,0 +1,24 @@
+import packageJson from '../package.json';
+
+export const MICROBLOCKS_ENABLED = true;
+
+export const IS_BROWSER = typeof document !== 'undefined';
```

Comment:
Let's make this a fn rather than const, in another branch i had

`function isClient() {}`

Reply:
removed for now, will add later if necessary

Reply:
> Let's make this a fn rather than const, in another branch i had

in what branch?